tests_async/system/credentials.json.enc

```diff
@@ -0,0 +1,52 @@
+U2FsdGVkX1+wqu1+eVu6OPbPoE0lzIp3B11p8Rdbha1ukxXcsskegJdBjcUqQOav
```

Comment:
We could probably do without duplicating these files, instead importing them from the main tests directory, but it's up to you.

Reply:
I am going to make a process issue to cover this work. I don't want to stretch this PR out if possible :)

Reply:
https://github.com/googleapis/google-resumable-media-python/issues/181